src/pages/Home.page.tsx

```
@@ -1,5 +1,195 @@
+      {row.map((space, spaceIndex) => (
+        <Paper
+          component={Stack}
+          align='center'
+          justify='center'
+          radius={0}
+          onClick={() => {
+            onSpaceClick({ spaceIndex });
+          }}
+          bg={determineSpaceColor({ rowIndex, spaceIndex })}
+          h={getSquareSize(dimensions)}
+          w={getSquareSize(dimensions)}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)